.travis.yml

```diff
@@ -11,8 +11,9 @@ stages:
   - name: Lint
   - name: Build
   - name: Test
-  - name: Publish
-    if: branch = master OR commit_message = "[publish]"
+  - name: Publish_Chart
```

Comment:
I would suggest adding the conditions back for this stage otherwise you end publishing the chart on every feature branch which overwrites the one in incubator generated by master.

You can still publish a chart from a feature branch but it generates an "A" version so the chart from master is not overwritten in incubator.

Reply:
Based on Sergiu's implementation of travis file, this will never happen as master will only publish on stable.

Reply:
That's not quite true, master publishes the chart to incubator by default (line 23) and only publishes to stable if the branch is master and commit message is "[release]" (line 78).

Reply:
That's correct. Somehow I was reading the `&&` in the if statement as an `||` :-) I'll fix it now.

Reply:
Done!